src/rust/ensogl/src/display/camera/camera2d.rs

```diff
@@ -79,7 +79,7 @@ pub struct Clipping {
 impl Default for Clipping {
     fn default() -> Self {
         let near = 1.0;
-        let far  = 1000.0;
+        let far  = 10000.0;
```

Comment:
add todo comment that this is not a final solution (will fail on bigger screens). Instead, when resizing a screen this should be updated - write about ti in the todo pls